varats/tools/driver_paper_config.py

```diff
@@ -44,12 +45,14 @@ def main() -> None:
     # vara-pc set
     set_parser = sub_parsers.add_parser('select',
                                         help="Select the current paper config.")
-    set_paper_config_parser_arg(set_parser)
+    set_paper_config_parser_arg(set_parser, True)
 
     # vara-pc list
     list_parser = sub_parsers.add_parser(
         'list', help="List all available paper configs")
-    set_paper_config_parser_arg(list_parser, True)
+    list_parser.add_argument("--paper_config_path",
```

Comment:
```suggestion
    list_parser.add_argument("--paper-config-path",
```
All other optional arguments are written with dashes. The parser takes care of converting them to underscores.

Reply:
I can change this if you like but the initial option `paper_config` that you added is written like this.
I wondered about this yesterday when moving this to the function but assumed you did this on purpose.